src/components/HacksPage.js

```diff
@@ -36,18 +32,17 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const Hacks = ({ dispatch }) => {
+export const Hacks = () => {
   const classes = useStyles();
   const [data, setData] = useState([]);
 
   useEffect(() => {
     const fetchData = async () => {
-      let result = await axios.get(process.env.REACT_APP_API_URL + "hacklist");
-      //console.log(result.data);
+      let result = await axios.get(UrlJoin(process.env.REACT_APP_API_URL, "hacklist"));
       setData([...result.data]);
     };
     fetchData();
-
+  });
```

Comment:
don't you still want an empty dependencies array here? otherwise you will refetch the hacks after every successful render (which means after the setData call... have a look at your network tab in chrome dev tools to see if your remaking this request repeatedly) https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect